### ISSUE TITLE
Report shows incorrect user shutdown scenario

### ISSUE THREAD

**Issue by EugeneAbramchuk** [gatling/gatling#3338]
Hey guys,

It looks like there is a problem with the report generation. In my scenario I start 250 users and 236 remain active until after the scenario ends (3600 sec duration). However the report shows that all users stopped in the middle of the test, while the scenario output and monitoring clearly contradict it. 

The last user amount before they started falling down is the correct value of 236.

Gatling version 2.2.5

<img width="914" alt="screen shot 2017-08-02 at 15 36 44" src="https://user-images.githubusercontent.com/14140360/28874548-012d7d30-779b-11e7-90cf-9a0911001c3a.png">
<img width="639" alt="screen shot 2017-08-02 at 15 36 17" src="https://user-images.githubusercontent.com/14140360/28874556-0899024c-779b-11e7-9a03-9915605e47ac.png">

**Comment by slandelle** [comment]
The number of concurrent users is calculated as:

```
the number of concurrent users in the last bucket
+ the number of users that were started during this bucket
- the number of users stopped during this bucket
```

Either there's a bug, or your users die very very fast.
Could you please provide your simulation.log?

**Comment by EugeneAbramchuk** [comment]
Yes, sure.

p.s. the users are dying due to the problem I described in #3336. Seeing the report bug, it's clear now that all the users die after receiving a non-HTTP 200 OK response from the Amazon's load balancer.

[simulation.log.zip](https://github.com/gatling/gatling/files/1193918/simulation.log.zip)

**Comment by slandelle** [comment]
You got me wrong, I want the simulation.log file generated in your target dir, along with the reports, not the console output.

**Comment by EugeneAbramchuk** [comment]
My apologies
[simulation.log.zip](https://github.com/gatling/gatling/files/1194040/simulation.log.zip)

**Comment by slandelle** [comment]
The reports generated on my side from  your simulation.log don't match your screenshots:

<img width="901" alt="capture d ecran 2017-08-02 a 16 33 26" src="https://user-images.githubusercontent.com/849197/28878741-764eeafe-77a0-11e7-9821-da5f124f42c1.png">

I'm pretty sure you're not actually running Gatling 2.2.5, but an older version.
Your issue is most likely #3220 that's been fixed in 2.2.5.

**Comment by EugeneAbramchuk** [comment]
You are right, the test checked out of the branch with the Gatling 2.2.4. Thanks!
